beta/src/content/apis/react/useDebugValue.md

```diff
@@ -1,22 +1,125 @@
 ---
-title: useDebugValue
+title: useContext
```

Comment:
We are not using the title anywhere? and this should be `useDebugValue`

Reply:
i think i fixed that after merge